Tests/XcodeprojTests/GenerateXcodeprojTests.swift

```diff
@@ -29,15 +29,18 @@ class GenerateXcodeprojTests: XCTestCase {
     func testXcodebuildCanParseIt() {
       #if os(macOS)
         mktmpdir { dstdir in
-            let fileSystem = InMemoryFileSystem(emptyFiles: "/Sources/DummyModuleName/source.swift")
+            let packagePath = dstdir.appending(component: "foo")
+            let modulePath = packagePath.appending(components: "Sources", "DummyModuleName")
```

Comment:
Not an issue, but `func appending(components names: String...) -> AbsolutePath` is an interesting implementation... 🤔

Reply:
We use that function all over the place though.  Are you saying it would be better to use two appends?